a3_finance/a3_finance/doc_events/payroll_entry.py

```
@@ -82,3 +82,74 @@ def get_employees_by_employment_type(payroll_entry_name, employment_types: str):
     doc.save()
 
     return doc
+
+
+
+import frappe
+from hrms.payroll.doctype.payroll_entry.payroll_entry import PayrollEntry
```

Ruff (0.14.5)

Redefinition of unused `PayrollEntry` from line 3

Remove definition: `PayrollEntry`

(F811)